docs/src/Callout.doc.js

```diff
@@ -267,7 +451,53 @@ function Example(props) {
     </Box>
   );
 }
-`}
+        `}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Dismissible"
+      description={`
+        A \`dismissButton\` can be used when a Callout does not indicate a persistent state. This will most commonly be used in info Callouts.
+
+        Do not use dismiss buttons in the following cases:
+
+        - There is a persistent account or page status that the user must address.
+        - The user must access this information again in order to perform a task.
+        `}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Callout
+  type="info"
+  iconAccessibilityLabel="Info"
+  title="Your business account was created!"
+  message="Apply to the Verified Merchant Program"
+  primaryAction={{href: "https://pinterest.com", label: "Get started", accessibilityLabel:"Get started: Verified Merchant Program"}}
+  secondaryAction={{href: "https://pinterest.com", label: "Learn more", accessibilityLabel:"Learn more: Verified Merchant Program"}}
+  dismissButton={{
+    accessibilityLabel: 'Dismiss this banner',
+    onDismiss: ()=>{},
+  }}
+/>
+      `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Related"
+    description={`
+      [Upsell](/upsell)
+
+      If marketing new products or features, or encouraging upgrades, use an Upsell instead.
+
+      [Toast](/toast)
+
+      Toast provides feedback on a user interaction, like a confirmation that appears when a Pin has been saved. Unlike Upsell and Callout, Toasts don’t contain actions. They’re also less persistent, and typically disappear after a certain duration.
+    `}
```

Comment:
How do we feel about adding `ActivationCard` here? I feel ActivationCard could be more relevant than Toast.